packages/modeling/src/measurements/measureArea.js

```diff
@@ -23,10 +23,12 @@ const measureAreaOfPath2 = () => 0
  * @returns {Number} area of the geometry
  */
 const measureAreaOfGeom2 = (geometry) => {
+  if (geometry.area) return geometry.area
```

Comment:
Is this caching dangerous at all?  If the geometry is altered at all, does area either get invalidated or recalculated?

Reply:
A fair question. All operations on geometry produce a new object, as each object is considered immutable. For example, transform() creates a new object with a new matrix and a copy of the polygons. Everything else is discarded, i.e. the measurements.

Reply:
Good point. 👍